app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -0,0 +1,56 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::DetailedGuide < PublishingApiPresenters::Edition
+  def links
+    extract_links([
+      :lead_organisations
+    ]).merge(
+      related_guides: related_guides,
+      related_mainstream: related_mainstream
+    )
+  end
+
+private
+
+  def document_format
+    "detailed_guide"
+  end
+
+  def details
+    super.merge(
+      body: body,
+      first_public_at: first_public_at,
+      change_history: item.change_history.as_json,
+      related_mainstream_content: related_mainstream
+    )
+  end
+
+  def body
+    Whitehall::GovspeakRenderer.new.govspeak_edition_to_html(item)
+  end
+
+  def first_public_at
+    if item.document.published?
+      item.first_public_at
+    else
+      item.document.created_at.iso8601
+    end
+  end
+
+  def related_guides
+    item.try(:related_detailed_guide_ids) || []
```

Comment:
Let's fix+test and add the behaviour separately, in the interest of merging this ASAP.

I'm going to change it to return just `[]` so it's clear that it's unimplemented.